connectors/vault/vault_utils/vault_utils.go

```diff
@@ -135,3 +137,24 @@ func (vlt *VaultConnection) GetFromVault(innerVaultPath string) (string, error)
 
 	return string(b), nil
 }
+
+// GetFromVault returns the credentials from vault as json
```

Comment:
comment should start with `AddToVault`

Reply:
the content also should be changed to "adding" credentials instead of "getting" them

Reply:
fixed. Thanks.